packages/calendar/src/LionCalendar.js

```diff
@@ -223,6 +223,11 @@ export class LionCalendar extends LocalizeMixin(LitElement) {
     }
   }
 
+  reset() {
+    this.centralDate = this.__initialCentralDate;
+    this._requestUpdate('centralDate');
```

Comment:
Is this line needed? I would expect _requestUpdate to be called under the hood by UpdatingMixin from `lit-element`, since `centralDate` is already part of `static get properties`?